### DISCUSSION TITLE
[RAPIOF] Could not connect to release API

### DISCUSSION THREAD

**Discussion by zsewa** [TUM-Core-Facility-Microbiome/IMNGS2-Core-docs#3] [Error Codes]
**Background of this error**
The application queries a release endpoint for information about available releases of an environment (e.g., TIC). This query failed.

**Things to try**
- This is a know issue of versions below 0.1.6 as the authentication method of the endpoint changed. Please update to a version above 0.1.6.
- Please make sure you are connected to the internet
- If you are using non-official release channels, make sure you are pointing to the correct repositories

**Comment by annasaax-rgb** [comment]
Hi, I’m getting this same error even though my internet connection is working.

The app fails when trying to query available releases for the environment (TIC/TACTIC) and shows:

“Failed to query available releases”

I’m using the default setup (no custom release channels), so it should be pointing to the official repository.

It looks like the release endpoint might be incorrect or not reachable from my setup (WSL/conda).

Any help would be appreciated.

**Reply by zsewa** [reply]
Hi asrgb, this is a known issue with versions below 0.1.6, as the release endpoint changed how it authenticates downloads. Please upgrade to version 0.1.6-beta-1 where this is fixed. Sorry for the inconvenience.
Download link for v0.1.6-beta-1: https://drive.google.com/drive/folders/1hPCj6wPJ_hsyTD5lck0YNqfCapnoAGe2?usp=drive_link
